docs/guide/limitation_reference.md

```diff
@@ -0,0 +1,349 @@
+# Limitation reference
+
+## BlockingLimitation
+
+A generic Limitation type to use when no other Limitation has been implemented.
+Without any Limitation assigned, a `LimitationNotFoundException` is thrown.
+
+It is called "blocking" because it will always tell the permissions system that the User does not have access to any Policy it is assigned to, making the permissions system move on to the next Policy.
```

Comment:
```suggestion
It is called "blocking" because it will always tell the permissions system that the User does not have access to any Policy they are assigned to, making the permissions system move on to the next Policy.
```